html/lesson4/tutorial.md

```diff
@@ -350,6 +350,232 @@ Add space around the images so there is some separation between them and the tex
 }
 ```
 
+## Flexible Box Layout
+
+It is a CSS module that defines a set of properties for positioning, aligning, and distributing space between elements in a container, even if their size is unknown or dynamic.
+
+The basic idea is to separate items into a block container (`flex container`) and its children (`flex items`).
+
+![](assets/images/flex-container.svg)
+
+![](assets/images/flex-items.svg)
```

Comment:
Am I right in thinking that these images are copied from https://css-tricks.com/snippets/css/a-guide-to-flexbox/ ? Do we have permission to use them? This might not be legal usage. Personally I'd suggest linking to that guide instead of copy-pasting it here.

cc @KimberleyCook

Reply:
@richardwestenra, we can remove them and link the guide as a helpful link. I saw many other websites using those images as well. 

I can probably draw it by myself in the separate PR.